server/model/block.go

```diff
@@ -58,9 +58,13 @@ type Block struct {
 	// required: false
 	DeleteAt int64 `json:"deleteAt"`
 
-	// The workspace id that the block belongs to
+	// Deprecated. The workspace id that the block belongs to
+	// required: false
+	WorkspaceID string `json:"-"`
```

Comment:
Is there any reason to keep this here?

Reply:
Yes, we need it for the unique IDs data migration

Reply:
Shouldn't we replicate that in the migration code and leave it as is the real value now? I think we shouldn't contaminate the rest of the code with migration code details.

Reply:
You mean creating a `LegacyBlock` type that contains the old field? I'm happy to go that way